save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/vulnerability/VulnerabilityController.kt

```diff
@@ -221,6 +221,23 @@ class VulnerabilityController(
         ResponseEntity.ok("Vulnerability was successfully approve")
     }
 
+    @PostMapping("/reject")
+    @Operation(
+        method = "POST",
+        summary = "Reject vulnerability.",
+        description = "Reject vulnerability.",
+    )
+    @ApiResponse(responseCode = "200", description = "Successfully rejected vulnerability")
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
+    fun reject(
+        @RequestBody vulnerabilityDto: VulnerabilityDto,
+    ): Mono<StringResponse> = blockingToMono {
+        vulnerabilityService.reject(vulnerabilityDto)
+    }.map {
+        ResponseEntity.ok("Vulnerability was successfully rejected")
+    }
+
```

Comment:
To my mind, this method needs `@PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")` annotation